pythonforandroid/bootstraps/qt/build/blacklist.txt

```diff
@@ -0,0 +1,84 @@
+# prevent user to include invalid extensions
+*.apk
+*.aab
+*.apks
+*.pxd
+
+# eggs
+*.egg-info
+
+# unit test
+unittest/*
+
+# python config
+config/makesetup
+
+# unused kivy files (platform specific)
+kivy/input/providers/wm_*
+kivy/input/providers/mactouch*
+kivy/input/providers/probesysfs*
+kivy/input/providers/mtdev*
+kivy/input/providers/hidinput*
+kivy/core/camera/camera_videocapture*
+kivy/core/spelling/*osx*
+kivy/core/video/video_pyglet*
+kivy/tools
+kivy/tests/*
+kivy/*/*.h
+kivy/*/*.pxi
```

Comment:
Maybe needs to be cleaned up?

Reply:
right. coped from the other bootstraps. I guess, the kivy related files don't make sense. I will leave the rest as it is, as they come from built Python interpreter?